apps/web/src/pages/templates/components/VariantsPage.tsx

```diff
@@ -85,10 +228,21 @@ export function VariantsPage() {
             variant={step}
             nodeType="variantRoot"
             data-test-id="variant-root-card"
+            errorMessage={
+              errorState?.variantIndex === variants.length
+                ? errorState.errorMessage
+                : missingProviderError?.errorMsg ?? variantsErrorMap?.[`variant-${variants.length}`]
+            }
+            isActiveError={errorState?.variantIndex === variants.length}
+            nodeErrorType={errorState?.variantIndex === variants.length ? errorState.errorType : undefined}
```

Comment:
same as above, just the addition of missing provider error being taking the priority